src/rgw/rgw_sal_motr.cc

```diff
@@ -4498,8 +4489,7 @@ int MotrMultipartWriter::complete(size_t accounted_size, const std::string& etag
                            head_obj->get_bucket()->get_owner()->get_id().to_str(),
```

Comment:
pls, do add comments above this update call as well.

Reply:
done